app/portainer/views/endpoints/edit/endpointController.js

```diff
@@ -204,6 +205,24 @@ function EndpointController(
         }
       }
     );
+
+    // AMT info...
+    EndpointService.updateEndpoint(endpoint.Id, payload).then(
```

Comment:
is this ok? why the duplicate `updateEndpoint` ?

Reply:
:+1: don't let newbs commit exploring code